src/riak_kv_console.erl

```diff
@@ -548,15 +550,15 @@ bucket_type_print_create_result(Type, ok) ->
         false ->
             ok
     end;
-bucket_type_print_create_result(Type, {error, {_LineNo, riak_ql_parser, Reason}}) ->
-    io:format("Error validating table definition for bucket type ~ts:\n~s\n", [Type, Reason]),
-    error;
```

Comment:
In this case it really does need to be printed to the console, maybe to console.log wouldn't hurt as well.